package.json

```diff
@@ -23,7 +26,6 @@
   },
   "scripts": {
     "build": "hardhat compile",
-    "prepare": "hardhat compile && tsc",
```

Comment:
The `prepare` step gets called automatically when you install Comet as a dependency. However, the `hardhat compile` command requires all of the environment variables that we require in `hardhat.config.ts`.

As far as I can tell, we're not actually using this command anywhere. Removing it allows us to include Comet as a dependency in Webb3